packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/CommunityStakeStep/SignStakeTransactions/SignStakeTransactions.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import { CWDivider } from 'views/components/component_kit/cw_divider';
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWButton } from 'views/components/component_kit/new_designs/cw_button';
+import ActionSteps from '../../../components/ActionSteps';
+import CreateCommunityHint from '../../../components/CreateCommunityHint';
+
+import './SignStakeTransactions.scss';
+
+interface SignStakeTransactionsProps {
+  onOptOutEnablingStake: () => void;
+}
```

Comment:
same `types.ts` suggestion

Reply:
per our conversation, we should create `types.ts` file if there is more than one type for file